learn/analytics/bind_events_user.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: Bind analytics events to a user
+description: This guide shows you how to manually bind analytics events to search queries using the X-MS-USER-ID HTTP header.
+---
+
+# Bind analytics events to a user
+
+By default, Meilisearch associates analytics events with the most recent search of the user who triggered them. This guide shows you how to manually bind analytics events to search queries using the `X-MS-USER-ID` HTTP header.
```

Comment:
```suggestion
By default, Meilisearch uses IP addresses to identify users and calculate the Total User metrics. This guide explains how to use the `X-MS-USER-ID` HTTP header to manually link analytics events to specific users. This is especially useful if you're searching from your backend, as all searches would otherwise appear to come from your server's IP address, making it difficult to accurately track the number of individual users.
```